frontend/model/errors.js

```diff
@@ -32,7 +32,15 @@ export class GIErrorUnrecoverable extends Error {
   }
 }
 
-// -----------------------
+export class GIErrorInvalidChatroomAction extends Error {
+  constructor (...params: any[]) {
+    super(...params)
+    this.name = this.constructor.name
+    if (Error.captureStackTrace) {
+      Error.captureStackTrace(this, this.constructor)
+    }
+  }
+}
```

Comment:
Could you remove this and replace all the errors and use it with `throw new Error(` instead?

Reply:
One question here.
When do we need to create a new Error object such as `GIErrorIgnoreAndBanIfGroup`?
Actually, I can see several Error objects in `frontend/model/errors.js` but they are perfectly the same except for the names of them.

Reply:
These errors will cause Group Income to do very different things because of how `state.js` works. Please just use regular `Error` for now, and if you're curious, see also recently opened issue #1165